docs/contents/services.md

```diff
@@ -125,12 +125,6 @@ Controller Release Roadmap](https://github.com/aws/aws-controllers-k8s/projects/
 * Next milestone: [`DEVELOPER PREVIEW`](https://github.com/aws/aws-controllers-k8s/milestone/10)
 * AWS service documentation: https://aws.amazon.com/lambda/
 
-## Amazon EFS
-
-* Proposed: https://github.com/aws/aws-controllers-k8s/issues/328
-* Current release status: `PROPOSED`
-* AWS service documentation: https://aws.amazon.com/efs/
-
```

Comment:
Any reason to remove the above? It's still listed in the table above...

Reply:
It was duplicated: https://github.com/aws/aws-controllers-k8s/pull/671/files#diff-d34969a303cc6cb28f7d748f100fc323faa0f65400880d06b8e3f4e902c5d268R83